app/views/organizations/_liquid.html.erb

```diff
@@ -12,7 +12,11 @@
     </div>
   </a>
   <div class="ltag__user__content">
-    <h2><a href="/<%= organization.slug %>" class="ltag__user__link"><%= organization.name %></a><%= follow_button %></h2>
+    <h2>
+      <a href="/<%= organization.slug %>" class="ltag__user__link"><%= organization.name %></a>
+      <%= follow_button %>
+      <span class="hidden screen-reader-only">Follow organization: <%= organization.second %></span>
```

Comment:
The issue with the failing tests is here - `organization.second` doesn't seem to be defined (so raises an error when called). 

What did you want the "Follow organization" span to read?